tests/foreman/ui/test_computeresource_rhev.py

```diff
@@ -644,9 +724,95 @@ def test_positive_provision_rhev_with_image(self):
+            parameter_list_img = [
+                ['Name', img_name],
+                ['Operatingsystem', self.rhev_img_os],
+                ['Architecture', self.rhev_img_arch],
+                ['Username', self.rhev_img_user],
+                ['Password', self.rhev_img_pass],
+                ['uuid', self.rhev_img_name],
+            ]
+            self.compute_resource.add_image(cr_name, parameter_list_img)
+            imgs = self.compute_resource.list_images(cr_name)
+            self.assertTrue(img_name in imgs)
+            self.compute_resource.set_profile_values(
+                cr_name, COMPUTE_PROFILE_LARGE,
+                cluster=self.rhev_datacenter,
+                cores=2,
+                memory=1024,
+                image=self.rhev_img_name,
+                network_interfaces=[
+                    dict(
+                        name='nic1',
+                        network=settings.vlan_networking.bridge
+                    ),
+                ],
+                storage=[
+                    dict(
+                        size='10',
+                        storage_domain=self.rhev_storage_domain,
+                        bootable=True,
+                        preallocate_disk=False
+                    ),
+                ]
+            )
+            make_host(
+                session,
+                name=hostname,
+                org=ANY_CONTEXT['org'],
+                loc=self.loc_name,
+                parameters_list=[
+                    ['Host', 'Organization', self.org_name],
+                    ['Host', 'Location', self.loc_name],
+                    ['Host', 'Host group', self.config_env['host_group']],
+                    ['Host', 'Deploy on', cr_name],
+                    ['Host', 'Compute profile', COMPUTE_PROFILE_LARGE],
+                    ['Host', 'Puppet Environment',
+                        self.config_env['environment']],
+                    ['Operating System', 'Operating System', self.rhev_img_os],
+                    ['Operating System', 'Partition table',
+                        self.config_env['ptable']],
+                    ['Operating System', 'PXE loader', 'PXELinux BIOS'],
+                    ['Operating System', 'Root password', root_pwd],
+                ],
+                provisioning_method='network'
+            )
+            self.assertIsNotNone(self.hosts.search(
+                u'{0}.{1}'.format(hostname.lower(),
+                                  self.config_env['domain']
+                                  )))
+            host_ip = entities.Host().search(query={
+                'search': 'name={0}.{1} and organization="{3}"'.format(
+                    hostname.lower(),
+                    self.config_env['domain'],
+                    self.org_name)
+                    })[0].read().ip
+            result, stdout = self.compute_resource.host_provisioning_check(
+                                host_ip)
+            self.assertIs(result, 0, stdout)
```

Comment:
if host_provisioning_check will fail it will raise exception, this assert is reached only in the case of success check
see my my comment  about  ProvisioningCheckError
and lines 809-811 will became
```python
with self.assertNotRaises(ProvisioningCheckError):
    self.compute_resource.host_provisioning_check(host_ip)
```